server/py/services/api/crud/model_monitoring/deployment.py

```diff
@@ -948,6 +968,66 @@ def _get_function_summary_infra(
                 )
         return function_summaries_list, base_period
 
+    async def _enrich_with_stream_stats(
+        self,
+        function_summaries: typing.Optional[
+            list[mlrun.common.schemas.model_monitoring.FunctionSummary]
+        ],
+        agg_stats: bool = True,
+    ):
+        """
+        Enrich the function with stream stats.
+        :param function_summaries: List of `FunctionSummary` objects to enrich with stream stats.
+        :param agg_stats: If True, aggregate the stream stats by function name.
+        """
+
+        if isinstance(
+            self._stream_profile, mlrun.datastore.datastore_profile.DatastoreProfileV3io
+        ):
```

Comment:
```suggestion
        if not isinstance(
            self._stream_profile, mlrun.datastore.datastore_profile.DatastoreProfileV3io
        ):
           return
```

Reply:
It's correct that currently we only support `v3io` but in a few days we’ll add another PR that incldues `kafka` support as well. Therefore, it makes more sense to keep the current structure and later extend it with an `else`, something like:
```
        if isinstance(
            self._stream_profile, mlrun.datastore.datastore_profile.DatastoreProfileV3io
        ):
            # V3IO
            async with framework.utils.clients.async_nuclio.Client(
                self.auth_info
            ) as client: ...
       else:
           # kafka 
            import kafka...
```

Reply:
@Eyal-Danieli got it 👍